pom.xml

```diff
@@ -321,7 +321,7 @@
       <dependency>
         <groupId>commons-io</groupId>
         <artifactId>commons-io</artifactId>
-        <version>2.1</version>
+        <version>2.4</version>
```

Comment:
increased the version to fix
`java.lang.NoClassDefFoundError: org/apache/commons/io/Charsets`